spec/beaker-hostgenerator/generator_spec.rb

```diff
@@ -98,12 +57,20 @@ module BeakerHostGenerator
             fixture_hash['environment_variables'].each do |key, value|
               ENV[key] = value
             end
+
             cli = BeakerHostGenerator::CLI.new(arguments)
-            test_hash = YAML.load(cli.execute)
+            if fixture_hash['expected_exception']
+              # Turn fully-qualified classname string into an actual Class object.
+              expected_class = eval fixture_hash['expected_exception']
```

Comment:
`eval`!?!?!  Turns out Ruby doesn't have the most convenient way of turning a string classname into actual Class.  It doesn't support nesting of classes, so you have to walk the module/class hierarchy yourself, like this:
```ruby
        qualified_classname = fixture_hash['expected_exception'].split('::')
        expected_class = qualified_classname.reduce(Object) do |accum, e|
          accum.const_get(e)
        end
```

I opted to go with `eval` instead for simplicity sake.